### PR TITLE
[AAQ-456] Add tests and final cleanups

### PR DESCRIPTION
**Pull request by lickem22**
Reviewer: @suzinyou  
Estimate: 1h

---

## Ticket

Fixes: [AAQ-456](https://idinsight.atlassian.net/browse/AAQ-456)

### Goal
The goal of this PR is to update the tests with multilingual support and do the final cleanup/ tests to the frontend to make sure the  first changes work seamlessly.  
### Changes
The following changes have been made on the front end:
  - Fixing the snackmessage not displaying when deleting last content
  - Fixing error when deleting last content
 The following changes have been made to backend:
 - Updating tests
 - Updating validation tests

## How has this been tested?
This has been tested by running test and making sure it is working fine. This is the final feature PR. I suggest testing by making sure everything works as it should. If an issue is noticed we can decide whether to fix right away or raise the ticket depending on how critical it is and how easy it is to fix. 
## Checklist

Fill with `x` for completed. Delete any lines that are not relevant

- [x] My code follows the style guidelines of this project
- [x] I have reviewed my own code to ensure good quality
- [x] I have tested the functionality of my code to ensure it works as intended
- [x] I have resolved merge conflicts
- [x] I have updated the automated tests (if applicable)
- [ ] I have updated the requirements (if applicable)
- [ ] I have updated the README file (if applicable)
- [ ] I have updated affected documentation (if applicable)

[AAQ-456]: https://idinsight.atlassian.net/browse/AAQ-456?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: admin_app/package.json [hunk 1]
```diff
@@ -28,4 +28,4 @@
     "prettier-plugin-organize-imports": "^3.2.4",
     "typescript": "^5"
   }
-}
+}
```
Comment (lickem22) [inline admin_app/package.json:31]: Can you please fix that again on your side please @suzinyou

**Comment by suzinyou** [comment]
It's weird that unittests GA didn't run!

**Comment by suzinyou** [comment]
Oh weird. It ran, but on the merge commit, and it's not getting associated with this PR 🤔 
https://github.com/IDinsight/aaq-core/actions/runs/8615477433

**Review by suzinyou — APPROVED** [review]
Works as expected! Did the validation script run?

File: admin_app/src/app/content/page.tsx [hunk 2]
```diff
@@ -314,6 +315,7 @@ const CardsGrid = ({
                         return apiCalls.deleteContent(content_id, null, token!);
                       }}
                       deleteLanguageVersion={handleDeleteLanguageVersion}
+                      setRefreshKey={setRefreshKey}
```
[the remark this answers is not in the document]
Comment (lickem22) [reply admin_app/src/app/content/page.tsx:318]: It refreshes the content after deleting contents

**Pull request merged**